policyengine_us/variables/gov/states/id/tax/income/credits/grocery/id_grocery_credit.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class id_grocery_credit(Variable):
```

Comment:
create a id_grocery_credit_eligible.py 

with the formula:

snap_ineligible = snap > 0
incarcerated = person("is_incarcerated", period)

return snap_ineligible & ~incarcerated

Reply:
need a test for this new variable